fe/fe-core/src/main/java/com/starrocks/alter/OptimizeJobV2.java

```diff
@@ -396,6 +397,7 @@ protected void runRunningJob() throws AlterCancelException {
         this.finishedTimeMs = System.currentTimeMillis();
 
         GlobalStateMgr.getCurrentState().getEditLog().logAlterJob(this);
+        WarehouseIdleChecker.updateJobLastFinishTime(warehouseId);
```

Comment:
Is it a better choice to put it in AlterJobV2::run? Then you don't need to write each job separately.

Reply:
done.